### ISSUE TITLE
[WORK] SwinTransformer

### ISSUE THREAD

**Issue by dustywindow** [Dementia-Diagnosis-Project/memoria_AI#15]
# WORK PLAN 
~ 작업 내용 설명 ~ 
- [X] 모델 : Swin Transformer v2
  - 선정이유
    a. Computer Vision 분야에서 가장 최신 모델
    b. 
  - 사용 프레임워크
  - 데이터
 
- [X] Data Preprocessing

- [X]

### 학습방법
learning rate: 1e-4
batch size : 30
criterion: CrossEntropyLoss
optimizer: Adam

# ISSUES
### 문제상황
~ 문제 상황 설명 ~

### 예상 해결 방안 list 
- [ ] 해결 방안 1
- [ ] 해결 방안 2

### (해결시) 해결방법   
~ 해결방법/결과 설명 ~

# RESULTS
### Experiment 1
  - A fine-tuned version of [microsoft/swinv2-base-patch4-window16-256](https://huggingface.co/microsoft/swinv2-base-patch4-window16-256) on Alzheimer MRI data
  - Without Augmentation (only Resize 224*224 and Grayscale to RGB)
 
|Epoch|Step|Training Loss|Validation Loss|Accuracy|Specificity|Sensitivity|AUC|
|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
|1|411|0.7057|0.6788|테스트5|테스트6|테스트7|테스트8|
|2|822|0.6645|0.6436|테스트5|테스트6|테스트7|테스트8|
|3|1233|0.6355|0.6105|테스트5|테스트6|테스트7|테스트8|
|4|1644|0.6008|0.5542|0.717|0.742|0.692|0.717|
|5|2055|0.5289|0.4722|테스트5|테스트6|테스트7|테스트8|
|6|2466|0.3949|0.3426|테스트5|테스트6|테스트7|테스트8|
|7|2877|0.2836|0.2248|테스트5|테스트6|테스트7|테스트8|
|8|3288|0.2048|0.2051|테스트5|테스트6|테스트7|테스트8|
|9|3699|0.1428|0.1904|테스트5|테스트6|테스트7|테스트8|
|10|4110|0.1146|0.1205|테스트5|테스트6|테스트7|테스트8|
|11|4521|0.0849|0.1300|테스트5|테스트6|테스트7|테스트8|
|12|4932|0.0679|0.1411|테스트5|테스트6|테스트7|테스트8|
|13|5343|0.0665|0.1342|테스트5|테스트6|테스트7|테스트8|
|14|5754|0.0547|0.1218|테스트5|테스트6|테스트7|테스트8|
